hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/debug/TestContainerKeyScanner.java

```diff
@@ -0,0 +1,252 @@
+  private StringWriter stdout, stderr;
+  private PrintWriter pstdout, pstderr;
+  private CommandLine cmd;
+  private static final Gson GSON =
+      new GsonBuilder().setPrettyPrinting().create();
+  private static final ContainerKeyInfo KEY_ONE =
+      new ContainerKeyInfo(1L, "vol1", -123L, "bucket1", -456L, "dir1/key1",
+          -789L);
+  private static final ContainerKeyInfo KEY_TWO =
+      new ContainerKeyInfo(2L, "vol1", 0L, "bucket1", 0L, "key2", 0L);
+  private static final ContainerKeyInfo KEY_THREE =
+      new ContainerKeyInfo(3L, "vol1", 0L, "bucket1", 0L, "key3", 0L);
+
+  private static final Map<Long, List<ContainerKeyInfo>> CONTAINER_KEYS =
+      new HashMap<>();
+
+  static {
+    List<ContainerKeyInfo> list1 = new ArrayList<>();
+    list1.add(KEY_ONE);
+    List<ContainerKeyInfo> list2 = new ArrayList<>();
+    list2.add(KEY_TWO);
+    List<ContainerKeyInfo> list3 = new ArrayList<>();
+    list3.add(KEY_THREE);
+    CONTAINER_KEYS.put(1L, list1);
+    CONTAINER_KEYS.put(2L, list2);
+    CONTAINER_KEYS.put(3L, list3);
+  }
+
+  private static final ContainerKeyInfoResponse KEYS_FOUND_OUTPUT =
+      new ContainerKeyInfoResponse(3, CONTAINER_KEYS);
+
+  private static final String KEYS_NOT_FOUND_OUTPUT =
+      "No keys were found for container IDs: [1, 2, 3]\n" +
+          "Keys processed: 3\n";
+
+  @BeforeEach
+  public void setup() throws IOException {
+    OzoneConfiguration conf = new OzoneConfiguration();
+    stdout = new StringWriter();
+    pstdout = new PrintWriter(stdout);
+    stderr = new StringWriter();
+    pstderr = new PrintWriter(stderr);
+
+    cmd = new CommandLine(new OzoneDebug())
+        .addSubcommand(new ContainerKeyScanner())
+        .setOut(pstdout)
+        .setErr(pstderr);
+
+    dbStore = DBStoreBuilder.newBuilder(conf).setName("om.db")
+        .setPath(tempDir.toPath()).addTable(KEY_TABLE).addTable(FILE_TABLE)
+        .addTable(DIRECTORY_TABLE)
+        .build();
+  }
+
+  @AfterEach
+  public void shutdown() throws IOException {
+    closeDbStore();
+    pstderr.close();
+    stderr.close();
+    pstdout.close();
+    stdout.close();
+  }
+
+  @Test
+  void testWhenThereAreKeysForContainerIds() throws IOException {
+
+    // create keys for tables
+    long volumeId = -123L;
+    long bucketId = -456L;
+    long dirObjectId = -789L;
+    createDirectory(volumeId, bucketId, bucketId, dirObjectId, "dir1");
+    createFile(volumeId, bucketId, "key1", -987L, dirObjectId, 1L);
+    createKey("key2", 2L);
+    createKey("key3", 3L);
+
+    String[] cmdArgs =
+        {"ckscanner", "--om-db", dbStore.getDbLocation().getAbsolutePath(),
+            "--container-ids", "1 2 3"};
+
+    closeDbStore();
+
+    int exitCode = cmd.execute(cmdArgs);
+    Assertions.assertEquals(0, exitCode);
+
+    Assertions.assertEquals(
+        GSON.fromJson(stdout.toString(), ContainerKeyInfoResponse.class),
+        KEYS_FOUND_OUTPUT);
+
+    Assertions.assertTrue(stderr.toString().isEmpty());
```

Comment:
nit: please use `assertThat` for better failure message

```suggestion
    assertThat(stderr.toString()).isEmpty();
```

(needs `import static org.assertj.core.api.Assertions.assertThat`)

Reply:
Done.